Backpack/Common.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef BPK_Common_h
+#define BPK_Common_h
+
+#import <pthread.h>
+
+#if !defined(NS_BLOCK_ASSERTIONS)
+#define BPK_ASSERTIONS_ENABLED 1
+#else
+#define BPK_ASSERTIONS_ENABLED 0
+#endif
+
+#define BPKAssert(condition, desc, ...) NSAssert(condition, desc, ##__VA_ARGS__)
```

Comment:
This should also be guarded with a `BPK_ASSERTIONS_ENABLED` check if we do think we should have a custom assertion control?